docs/plugins/regrass.rst

```diff
@@ -5,15 +5,56 @@ regrass
     :summary: Regrow surface grass and cavern moss.
     :tags: adventure fort armok animals map
 
-This command can refresh the grass (or subterranean moss) growing on your map.
-It will only regrow the grass and moss on tiles that had some to begin with.
+This command can refresh the grass (and subterranean moss) growing on your map.
+Operates on floors, stairs, and ramps. Also works underneath shrubs, saplings,
+and tree trunks. Ignores furrowed soil, beaches, and tiles under buildings.
 
 Usage
 -----
 
 ::
 
-    regrass [max]
+    regrass [options]
 
-Specify the ``max`` keyword to pack more grass/moss onto a tile than what the
-game normally allows. This will give your grazers extra chewing time.
+Regrasses the entire map by default.
+
+``debugfilter set Debug regrass log`` can be used to figure out why regrass
+is failing on a tile. (Avoid regrassing the entire map with this enabled, as it
+will make the game unresponsive and flood the console for several minutes!)
+Disable with ``debugfilter set Info regrass log``.
+
+Options
+-------
+
+``h``, ``help``
+    Display this help screen.
+``m``, ``max``
```

Comment:
short options should be preceded by a single dash, long options with a double dash (getopt style)